psc-package/README.md

```diff
@@ -0,0 +1,99 @@
+# `psc-package`
+
+`psc-package` is an executable which helps manage PureScript dependencies via Git. It can be used directly, but it is also designed to be used by external tools.
+
+## Concepts
+
+### Package Sets
+
+A _package set_ is a mapping from package names to:
+
+- the Git repository URL for the package
+- the current version of the package
+- the package's transitive dependencies
+
+A package set repository contains a `packages.json` file which contains all mapping information. `psc-package` uses this information to decide which repos need to be cloned.
+
+The default package set is [purescript/package-sets](https://github.com/purescript/package-sets), but it is possible to create custom package sets by forking an existing package set or creating a new one from scratch. One benefit of using the default package set is that it is verified by a continuous integration process.
+
+## The `psc-package.json` format
+
+Here is a simple project configuration:
+
+```json
+{
+    "name": "my-project",
+    "set": "psc-0.10.2",
+    "source": "https://github.com/purescript/package-sets.git",
+    "depends": [
+        "prelude"
```

Comment:
@paf31 are the version bounds of the prelude dependency being explicitly dropped here? Or are they taken from the `bower.json` inside `my-project`?